test/lenses.js

```diff
@@ -41,16 +49,21 @@ describe('view, over, and set', function() {
 
   they('may be applied to composed lenses', function() {
     var streetLens = R.compose(addressLens, headLens);
+    var dogLens = R.compose(R.lensPath(['pets']), R.lensPath(['dog']));
+
+    eq(R.view(dogLens, alice), R.view(R.lensPath(['pets', 'dog']), alice));
 
     eq(R.view(streetLens, alice), '22 Walnut St');
 
     eq(R.over(streetLens, R.toUpper, alice),
        {name: 'Alice Jones',
-        address: ['22 WALNUT ST', 'San Francisco', 'CA']});
+        address: ['22 WALNUT ST', 'San Francisco', 'CA'],
+        pets: {dog: 'joker', cat: 'batman'}});
 
     eq(R.set(streetLens, '52 Crane Ave', alice),
        {name: 'Alice Jones',
-        address: ['52 Crane Ave', 'San Francisco', 'CA']});
+        address: ['52 Crane Ave', 'San Francisco', 'CA'],
+        pets: {dog: 'joker', cat: 'batman'}});
   });
-
```

Comment:
Can we keep this line? I'm sorry about the pedantry, but I do appreciate a tidy diff. :smile: